src/popup/App.vue

```diff
@@ -19,12 +19,16 @@
     >
       <Header v-if="showHeader" />
 
-      <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
-        <RouterView
-          :class="{ 'show-header': showHeader }"
-          class="main"
-        />
-      </transition>
+      <div class="app-container">
```

Comment:
Previously when user was scrolling the content it was going under the header. There is an idea of making the header semi-transparent to allow user to see the content that goes under the header. Following changes changed this behaviour.